client/src/views/OrgsList.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import makeStyles from '@mui/styles/makeStyles';
+import Card from '@mui/material/Card';
+import TodayOutlined from '@mui/icons-material/TodayOutlined';
+import RoomOutlined from '@mui/icons-material/RoomOutlined';
+import Grid from '@mui/material/Grid';
+
+import type { Theme } from '@mui/material/styles';
+
+import type { Asset } from '../types';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  needsAndOffersSub: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    flexWrap: 'wrap',
+    alignItems: 'stretch',
+  },
+  card: {
+    marginBottom: '30px',
+    '&:not(:last-child)': {
+      marginRight: '20px',
+    },
+    flex: '0 0 30%',
+  },
+  cardImg: {
+    borderRadius: '5px',
+    margin: '10%',
+    maxWidth: '80%',
+  },
+  needsAndOffersHeader: {
+    textAlign: 'left',
+    paddingBottom: '20px',
+  },
+  needsAndOffers: {
+    padding: '10%',
+  },
+  cardText1: {
+    padding: '0 10%',
+  },
+  cardText2: {
+    padding: '0 10% 10%',
+  },
+}));
+
+type Props = {
+  assets: Asset[];
+  headerContentRight?: JSX.Element;
+  headerText: string;
+};
+
+function AssetsList(props: Props): JSX.Element {
```

Comment:
component name should match filename
check volunteer list too

Reply:
I've duplicated them from AssetsList.tsx - slopy I didn't change the name. These are placeholders until we confirm how we want the cards in SearchResults to look.

OPINE: I believe since React is based on the single smallest component - the architecture here would benefit from having a single SearchResult window in which specialized cards exist (goods, services, volunteer ops) and are rendered based on characteristics within a combined list of all three. Just my 2 cents, but I may be incorrect as well.